client/windows/settings/BattleOptionsTab.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * BattleOptionsTab.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+#include "../../gui/InterfaceObjectConfigurable.h"
+#include "../../battle/BattleInterface.h"
```

Comment:
Prefer forward-declarations to includes in headers
```suggestion

class BattleInterface;
```

Reply:
done